query/src/main/java/org/infinispan/query/backend/QueryInterceptor.java

```diff
@@ -9,6 +9,8 @@
 import java.util.Set;
```

Comment:
This came from the "official" formatter unfortunately. Should we avoid in general to reformat comments?